### PR TITLE
#730: SKUs methods

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-white/issues/730 |
| Dependencies | https://github.com/ripe-tech/ripe-core/pull/4513 |
| Decisions | Methods for new [LCRUD SKU methods](https://github.com/ripe-tech/ripe-core/pull/4513). <br><br> Only missing the udpated documentation in `docs.platforme.com`. |
| Animated GIF |  image (1)|

### CODE REVIEW

**Review by Benedito0 — APPROVED** [review]

**Comment by gcandal** [comment]
@BeeMargarida is this ready? Please re-run the tests and undraft if it is.

File: test/js/api/sku.js [hunk 1]
```diff
@@ -6,6 +6,106 @@ const ripe = require("../../../src/js");
...
+                    },
+                    shadow: {
+                        material: "default",
+                        color: "default"
+                    }
+                })
+            );
+
+            // deletes the newly created SKU
+            result = await remote.deleteSkuP(createdSku.id);
```
Comment (gcandal) [inline test/js/api/sku.js:105]: try-finally

Comment (gcandal) [inline test/js/api/sku.js:105]: try-finally

**Comment by BeeMargarida** [comment]
@gcandal CI is failing because it can't find the new routes

**Review by gcandal — APPROVED** [review]

**Comment by BeeMargarida** [comment]
@gcandal Don't know if its supposed to be assigned to me

**Review by joamag — APPROVED** [review]

**Pull request merged**